.gitignore

```diff
@@ -125,5 +125,4 @@ dmypy.json
 # Project specific ignores
 bace/classifiers/neural/glove_embeddings/
 .idea/
-data/
-filtered_data/
```

Comment:
Why remove when you can just append "bace_data/" to the gitignore? Enforcing a file structure for how it would go in the gitignore is not a good idea in my opinion.

Reply:
alright, i'll do that